src/Modal/Modal.jsx

```diff
@@ -149,6 +154,7 @@ Modal.defaultProps = {
   cancelText: "",
   submitText: "",
   role: "dialog",
+  submissionEnabled: false,
```

Comment:
think a more sensible default might be true, seems like the most common use case (info prompts etc)